libafl/src/executors/forkserver.rs

```diff
@@ -644,6 +644,7 @@ where
     OT: ObserversTuple<S::Input, S>,
     S: UsesInput,
     SP: ShMemProvider,
+    TC: TargetBytesConverter,
```

Comment:
we dont't need this. we already have S, and is used for ObserversTuple<S::Input, S>. so input is just S::Input.

Reply:
ah ok it's necessary because this one is now moved out of executor

Reply:
We should add TargetBytesConverter support to command executor eventually